src/basic/socket-util.c

```diff
@@ -1319,6 +1320,52 @@ int flush_accept(int fd) {
         }
 }
 
+int flush_mqueue(int fd) {
+        _cleanup_free_ char *buf = NULL;
+        struct mq_attr attr;
+        unsigned pri;
+        int r, count = 0;
```

Comment:
Please use `ssize_t` for `count`, and make this return `ssize_t` rather than `int`.

Reply:
Done